core/network/fabric-interop-cc/contracts/interop/membership.go

```diff
@@ -9,45 +9,334 @@
 package main
 
 import (
+	"crypto/x509"
+	"encoding/base64"
 	"encoding/json"
+	"encoding/pem"
 	"fmt"
 
 	"github.com/hyperledger/fabric-contract-api-go/contractapi"
 	"github.com/hyperledger-labs/weaver-dlt-interoperability/common/protos-go/common"
+	"github.com/hyperledger-labs/weaver-dlt-interoperability/common/protos-go/identity"
+	protoV2 "google.golang.org/protobuf/proto"
 )
 
 const membershipObjectType = "membership"
+const membershipLocalSecurityDomain = "local-security-domain"
 
-func validateMemberCertChains(members map[string]*common.Member, securityDomain string) error {
-	for member, creds := range members {
-		if len(creds.Chain) > 1 {
-			err := verifyCertificateChain(nil, creds.Chain)
+// Check if the calling client has an attribute in its signing certificate indicating that it is a privileged network administrator
+//func isClientNetworkAdmin(stub shim.ChaincodeStubInterface) (bool, error) {
+func isClientNetworkAdmin(ctx contractapi.TransactionContextInterface) (bool, error) {
```

Comment:
Should we put these functions in `utils.go`? Since relay user checks are there.

Reply:
Yes, I'd intended to do that but forgot.